OracleDatabase/12.1.0.2/Vagrantfile

```diff
@@ -20,38 +20,91 @@ VAGRANTFILE_API_VERSION = "2"
 BOX_URL = "https://oracle.github.io/vagrant-projects/boxes"
 BOX_NAME = "oraclelinux/7"
 
-# define hostname
-NAME = "oracle-12102-vagrant"
-
 unless Vagrant.has_plugin?("vagrant-proxyconf")
   puts 'Installing vagrant-proxyconf Plugin...'
   system('vagrant plugin install vagrant-proxyconf')
 end
 
-# get host time zone for setting VM time zone, if possible
-# can override in env section below
-offset_sec = Time.now.gmt_offset
-if (offset_sec % (60 * 60)) == 0
-  offset_hr = ((offset_sec / 60) / 60)
-  timezone_suffix = offset_hr >= 0 ? "-#{offset_hr.to_s}" : "+#{(-offset_hr).to_s}"
-  SYSTEM_TIMEZONE = 'Etc/GMT' + timezone_suffix
-else
-  # if host time zone isn't an integer hour offset, fall back to UTC
-  SYSTEM_TIMEZONE = 'UTC'
+# Define constants
+Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
+  # Use vagrant-env plugin if available
+  if Vagrant.has_plugin?("vagrant-env")
+    config.env.load(".env.local", ".env") # enable the plugin
+  end
+
+  # VM name
+  NAME = default_s('NAME', 'oracle-12102-vagrant')
```

Comment:
I think we should probably call this one `VM_NAME` (or whatever prefix we decide on) to make it clear that it's the name of the VM itself.